projects/incident-language-model.md

```diff
@@ -0,0 +1,24 @@
+---
+remote_theme: nhsx/nhsx-io-theme
+title: Exploring large-scale language models with NHS incident data
+description: NHSX PhD Internship - Incident Language Model
+permalink: /incident-language-model/
+---
+
+# Exploring large-scale language models with NHS incident data
+
+**Keywords:** Natural Language Processing, Healthcare Text, Dataset, Embeddings, Clustering, Neural Networks
+
+**Need:** NHS England and NHS Improvement collect a national repository of incident reporting data for England and Wales.  These data are used to detect emerging patient safety incidents and drive learning that improves patient safety.  Although the as some categorical data fields, the real 'signals' in these data are within free-text field.  Clinical experienced teams review the most servious events, and emerging themes, but the scale of the dataset means that only a few percent can recieve a full review.  Natural Language Processing (NLP) techniques have the potential to unlock learning from data that are not currently reviewed.  The dataset is currently being used for topic modelling and other analyses, but a large scale language model is not yet in use.  The creation of an appropriate language model will allow the use of clustering and other methods to identify novel targets for clinical review.  This project would seek to build the first stages of a language model by applying different representations of text to find preferred models.
```

Comment:
`Although the as some categorical` -> `Although there are some categorical`
` Clinical experienced teams` -> ` Clinically experienced teams`
`currently reviewed` -> `currently fully reviewed`
`by applying different representations` -> `by constructing different representations`